### DISCUSSION TITLE
no matches for kind "KafkaRebalance" in version "kafka.strimzi.io/v1alpha1"

### DISCUSSION THREAD

**Discussion by vmm-afonso** [strimzi/strimzi-kafka-operator#8489] [General]
<img width="689" alt="MicrosoftTeams-image (2)" src="https://user-images.githubusercontent.com/115153075/236466277-7e827ca9-3774-4493-bd4f-af8065577bbc.png">

I'm following https://strimzi.io/blog/2020/06/15/cruise-control/, currently trying to ask CruiseControl to generate an optimization proposal. I'm getting the above error message after trying to apply the basic KafkaRebance yaml they provided in the documentation.
I checked and I have the right api-version and crd installed. Any idea what this could be?

**Comment by scholzj** [comment]
The blog post is almost 3 years old. It should give you an ide how Cruise Control works. But unless you use a Strimzi version from that time, then many things might have changed since. In this case, it is at least the API version.

You should check the examples on our GitHub and the docs corresponding to your version for the up-to-date features and examples. One thing which definitely changed is the `apiVersion` which would not be `kafka.strimzi.io/v1beta2`.

**Comment by vmm-afonso** [comment]
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/115153075/236474992-63094f08-30de-40a1-8a87-4942284a5bca.png)
Ok I guess that was the problem. It's a different api-version than the one they used in the strimzi cruise control doc.  Apologies, I think this thread can be closed now. Thank you for the help!
